packages/ui/alerts/alerts.jsx

```diff
@@ -66,18 +81,27 @@ export class FormAlerts extends React.PureComponent {
         {() => (<>
           <div className={styles.alertIcon}><Icon /></div>
           <div className={styles.alertText}>{content}</div>
-          <IconButton
-            Icon={Close}
-            className={styles.alertClose}
-            data-control-id="form-alert-close-button"
-            theme="tertiary"
-            onClick={this.onHideAlert(alert)}
-          />
+          {this.renderCloseBtn(persistent)}
         </>)}
       </Alert>
     );
   }
 
+  renderCloseBtn = (persistent) => {
```

Comment:
@ailincaibianca add `alert` as the argument to `renderCloseBtn` as it is needed for `onHideAlert`